test/Integration/utils/edge/testcontext.go

```diff
@@ -0,0 +1,28 @@
+/*
```

Comment:
the folder structure can be changed as : test/Integration/utils/edge/testcontext.go  -> test/integration/test_context.go
since we are already under kubeedge , it may be redundant to add one more "edge" sub-folder. also all go files can be named with an `_` in between if the name has multiple words.